scripts/populate-popular-and-new-and-noteworthy-projects.py

```diff
@@ -0,0 +1,140 @@
+"""
+This will update node links on POPULAR_LINKS_NODE and NEW_AND_NOTEWORTHY_LINKS_NODE.
+"""
+import sys
+import logging
+import datetime
+import dateutil
+import operator
+import requests
+from modularodm import Q
+from website.app import init_app
+from website import models
+from framework.auth.core import Auth
+from scripts import utils as script_utils
+from framework.transactions.context import TokuTransaction
+from website.settings.defaults import POPULAR_LINKS_NODE, NEW_AND_NOTEWORTHY_LINKS_NODE
```

Comment:
@pattisdr Right, these keys should be *defined* in `website.settings.default`, but when used they should be *imported* from `website.settings`, so that individual developers (i.e., e.g., me :) can override them locally in `website.settings.local`. No?